inst/report/comparison_report.Rmd

```diff
@@ -213,18 +227,18 @@ Cross-sectional estimates for HIV prevalence, ART coverage, and HIV incidence ar
 ```{r, echo = FALSE, results = 'asis'}
 
 if (print_en) {
-  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-baes survey(s): \n")
+  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-based survey(s): \n")
   cat("\n### Household Survey \n")
 }
 
 if (print_fr) {
-  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-baes survey(s): \n")
-  cat("\n### Household Survey \n")
+  cat("Ce rapport compare les estimations de Naomi pour ", quarter_t2, "à l'enquête de population suivante(s): \n")
+  cat("\n### Données d’enquête auprès des ménages \n")
 }
 
 if (print_pt) {
-  cat("This report compares Naomi estimates for ", quarter_t2, "to the following population-baes survey(s): \n")
-  cat("\n### Household Survey \n")
```

Comment:
This one too we didn't ask UNAIDS. I also updated to `population-based` that is correct? That was a typo?

Reply:
Will follow up with UNAIDS